src/index.js

```diff
@@ -0,0 +1,162 @@
+const sample = {
+
+  "basics": {
+    "name": "John Doe",
+    "label": "Programmer",
+    "picture": "",
+    "email": "john@gmail.com",
```

Comment:
I'd recommend keeping this in a separate file and then importing it - it makes it easier to work with and doesn't take up a huge amount of space in your code!

Reply:
For some reason, it kept giving me an error, saying that I can't use `require` or `import` to get the object. 

Definitely a must-do in the future.